base_jsonify/models/ir_exports_line.py

```diff
@@ -13,6 +13,29 @@ class IrExportsLine(models.Model):
         help="The complete path to the field where you can specify an "
         "alias on the a step as field:alias",
     )
+    lang_id = fields.Many2one(
+        comodel_name="res.lang",
+        string="Language",
+        help="If set, the language in which the field is exported",
+    )
+    resolver_id = fields.Many2one(
+        comodel_name="ir.exports.resolver",
+        string="Custom resolver",
+        help="If set, will apply the resolver on the field value",
+    )
+    function = fields.Char(
```

Comment:
so this would replace the callable feature in the parser?
As far as I see below, passing the callable in the simple parse is still allowed, right?

Reply:
Yes the existing callable feature is still fully supported (the corresponding test has not been modified). 
This "function" field is added so that your callable feature can also be used through the interface. Although you can only pass a method name, not a lambda. 
(I called it "function" instead of callable to avoid linter warnings and writing `if callable(callable)` stuff)

Reply:
I'd rename this to `instance_method_name` to make it clear.